packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -57,6 +57,10 @@
                 class="shrink-0 p-3 text-sm font-medium"
                 x-bind:class="{ 'bg-white @if (config('forms.dark_mode')) dark:bg-gray-800 @endif': tab === '{{ $tab->getId() }}' }"
             >
+                @if ($tab->hasIcon())
+                    <x-dynamic-component :component="$tab->getIcon()" class="h-5 w-5 mr-1 text-gray-500 @if (config('forms.dark_mode')) dark:text-gray-400 @endif inline"/>
```

Comment:
Probably cleaner to use `:class="\Illuminate\Support\Arr::toCssClasses([])"`

Reply:
Yeah, I don't think directives work inside of component attributes anyway?

Reply:
I prefer `Arr::toCssClasses()` in this context I think, directives look messy